js/index.js

```diff
@@ -101,8 +101,3 @@ modalTriggers.forEach((trigger) => {
     });
   });
 });
-
-const joinBtn = document.getElementById('btn-join');
-joinBTN.addEventListener('click', () => {
-  window.location.href = 'https://realdevsquad.com/join';
-});
```

Comment:
It's behind a feature flag, we can move ahead with the functionality of clicking the button
Check here :
https://realdevsquad.com/?join=true
https://realdevsquad.com

We only need to fix the functionality as mentioned in the issue, the click was not working

Reply:
done